packages/react-scripts/config/paths.js

```diff
@@ -80,13 +80,13 @@ module.exports = {
 if (__dirname.indexOf(path.join('packages', 'react-scripts', 'config')) !== -1) {
   module.exports = {
     appBuild: resolveOwn('../../../build'),
-    appPublic: resolveOwn('../template/public'),
-    appHtml: resolveOwn('../template/public/index.html'),
-    appIndexJs: resolveOwn('../template/src/index.js'),
+    appPublic: resolveOwn('../templates/app/public'),
```

Comment:
Yes, something like this. (You could also have /fixtures/kitchensink if you want to add more later.)